app/validators/url_validator.rb

```diff
@@ -0,0 +1,12 @@
+class UrlValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return if value.blank?
+    begin
+      uri = URI.parse(value)
+      record.errors.add(attribute, 'が正しい形式ではありません') unless uri.is_a?(URI::HTTP)
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->